tests/csm/rest/test_iam_users.py

```diff
@@ -4638,3 +4651,350 @@ def test_42272(self):
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        test_cfg = self.csm_conf["test_42273"]
+        resp_error_code = test_cfg["error_code"]
+        resp_msg_id = test_cfg["message_id"]
+        resp_data = self.rest_resp_conf[resp_error_code][resp_msg_id]
+        resp_msg_index = test_cfg["message_index_1"]
+        msg_1 = resp_data[resp_msg_index]
+        resp_msg_index = test_cfg["message_index_2"]
+        msg_2 = resp_data[resp_msg_index]
+        random_str = ''.join(secrets.choice(string.ascii_uppercase +
+                                            string.ascii_lowercase) for i in range(7))
+        special_str = ''.join(secrets.choice(string.punctuation) for i in range(7))
+        invalid_values = ["-1", "0", "0x89", random_str, special_str, "null", " "]
+        for key_value in invalid_values:
+            self.log.info("Testing for key value %s", key_value)
+            resp = self.csm_obj.list_iam_users_rgw(max_entries=key_value)
+            self.log.info("Verify Response : %s", resp)
+            assert_utils.assert_true(resp.status_code == HTTPStatus.BAD_REQUEST,
+                                     "Patch request status code failed")
+            if CSM_REST_CFG["msg_check"] == "enable":
+                self.log.info("Verifying error response...")
+                if key_value is "-1" or key_value is "0":
+                    assert_utils.assert_equals(resp.json()["error_code"], resp_error_code)
+                    assert_utils.assert_equals(resp.json()["message_id"], resp_msg_id)
+                    assert_utils.assert_equals(resp.json()["message"].lower(),
+                                           Template(msg_1).substitute(str_part="Max_entries").lower())
+                else:
+                    assert_utils.assert_equals(resp.json()["error_code"], resp_error_code)
+                    assert_utils.assert_equals(resp.json()["message_id"], resp_msg_id)
+                    assert_utils.assert_equals(resp.json()["message"].lower(),
+                                           Template(msg_2).substitute(A="Max_entries").lower())
+        self.log.info("##### Test completed -  %s #####", test_case_name)
+
+    @pytest.mark.lc
+    @pytest.mark.csmrest
+    @pytest.mark.cluster_user_ops
+    @pytest.mark.parallel
+    @pytest.mark.tags('TEST-42274')
+    def test_42274(self):
+        """
+        Test GET IAM user returns empty list with invalid  marker
+        """
+        test_case_name = cortxlogging.get_frame()
+        self.log.info("##### Test started -  %s #####", test_case_name)
+        test_cfg = self.csm_conf["test_42274"]
+        ran_int = test_cfg["ran_int"]
+        random_num = self.csm_obj.random_gen.randrange(1, ran_int)
+        random_str = ''.join(secrets.choice(string.digits +
+                                            string.ascii_lowercase) for i in range(7))
+        invalid_markers = [random_num, random_str, "null", '""']
```

Comment:
Move it to test config

Reply:
as per discussion, corrected the data types of list elements